app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -405,6 +422,8 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         final View menuBtn = view.findViewById(R.id.btn_menu);
         if (tabCounter != null) {
             tabCounter.setOnClickListener(this);
+            TabsSession tabsSession = TabsSessionProvider.getOrNull(getActivity());
+            tabCounter.setCount(tabsSession != null ? tabsSession.getTabsCount() : 0);
```

Comment:
I guess this should be placed in another commit? and there is `TabsSessionProvider.getOrThrow` in below, I think we should reuse it.

Reply:
OK, I will move it to another commit. Thank you!